File: extras python/extra2/Produto.py
class Produto:
    def __init__(self, nome, preco, idProduto):
        self.nome = nome
        self.preco = preco
        self.idProduto = idProduto
        pass
    
    #getters
    def getNome(self):
        return self.nome
    
    def getPreco(self):
        return self.preco
    
    def getIdProduto(self):
        return self.idProduto
    
    #setters
    def setNome(self, nome):
        self.nome = nome

    def setPreco(self, preco):
        self.preco = preco

    def setIdProduto(self, id):
        self.idProduto = id

    def __str__(self):
        return f'Nome: {self.nome}\n Preco: {self.preco}\n IdProduto: {self.idProduto}'

File: Experimentos/tabela/src/Produto.java
public class Produto {
    private String nome;
    private float preco;

    Produto(String nome, float preco){
        this.nome = nome;
        this.preco = preco;
    }

    //getters
    public String getNome() {
        return nome;
    }
    public float getPreco() {
        return preco;
    }

    //setters
    public void setNome(String nome) {
        this.nome = nome;
    }
    public void setPreco(float preco) {
        this.preco = preco;
    }
}
